conf/defaults.ini

```diff
@@ -1137,6 +1137,9 @@ promQueryBuilder = true
 # Experimental Explore to Dashboard workflow
 explore2Dashboard = true
 
+# Experimental Command Palette
+commandPalette = true
```

Comment:
Chatted with @davkal and we decided just having this available sooner is the right course of action, especially since it only adds functionality.